client/components/layout/Header.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <header>
+    <h1
+      v-if="$store.state.enableBack"
+      @click="() => $router.go(-1)"
+      class="back"
+    >
+      ←
+    </h1>
+    <h1> {{ $store.state.title }} </h1>
+    <section>
+      <article
+      v-for="(status, alert, index) in $store.state.alerts"
+      :key="index"
+      :class="status"
+      >
+        <p>{{ alert }}</p>
+      </article>
+    </section>
+  </header>
+</template>
+
+<style scoped>
+.back {
+  cursor: pointer;
+  text-align: center;
+  width: 3rem;
+  height: 3rem;
```

Comment:
line-height: 2.5rem; 

(this will center the arrow -> I checked with a variety of zoom levels, screen widths/heights, etc.)

without line-height: 
![image](https://user-images.githubusercontent.com/71414308/203666157-e7e6d801-84d0-4e5e-b9d5-ba8d2714fb7e.png)

with line-height: 
![image](https://user-images.githubusercontent.com/71414308/203666102-25c45ec3-5ea8-4a75-ac32-83faa90fea3e.png)

Reply:
interesting. In Chrome and Safari on my computer this is perfectly centered as far as I can tell and changing to 2.5 makes it very obviously not centered.

Reply:
this seems to be windows/mac specific. we'll deal with it later! :)